htsinfer/infer_read_orientation.py

```diff
@@ -22,5 +21,45 @@ def infer(
         LIBTYPE string according to Salmon documentation, cf.
         https://salmon.readthedocs.io/en/latest/library_type.html
     """
+    #create .tmp directory in "inputDir" and retuns .tmp path
+    tmpDir = mkTmpDir(inputDir)
+    
+    # call functions of issues #25, 26, 27 and 34
+    
+    # delete .tmp
+    rmTmpDir(tmpDir)
+        
     # implement logic
     return "U"
+    
+
+def mkTmpDir(inputDir):
```

Comment:
Make sure to add type hints to any input params of your functions as well as to the return value. In this case, your signature would become:

```py
def mk_tmp_dir(input_dir: str) -> str:
```

because you are expecting a string input and will return a string output